frontend/src/app/modules/auth/components/sign-up-form/sign-up-form.component.ts

```diff
@@ -15,6 +22,8 @@ export class SignUpFormComponent {
 
     public hidePassword = true;
 
+    private unsubscribe$ = new Subject<void>();
+
```

Comment:
Why did you add this?

Reply:
Yes, but you need to use this from BaseComponent

Reply:
Done